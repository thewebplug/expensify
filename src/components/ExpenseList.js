import React from 'react';
import ExpenseListItem from './ExpenseListItem';
const ExpenseList = () => (
    <div>
      <ExpenseListItem />
    </div>
)

export default ExpenseList;
