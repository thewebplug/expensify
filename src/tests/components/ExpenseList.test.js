import React from 'react';
import ExpenseList from '../../components/ExpenseList';
import { shallow } from 'enzyme';


test('should return ExpenseList with ExpenseListItems', () => {
    const wrapper = shallow(<ExpenseList />);
    expect(wrapper).toMatchSnapshot();
})
