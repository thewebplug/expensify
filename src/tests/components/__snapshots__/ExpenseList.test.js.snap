// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return ExpenseList with ExpenseListItems 1`] = `
<div>
  <Connect(ExpenseListItem) />
</div>
`;
